metabulo/views.py

```diff
@@ -177,6 +177,17 @@ def get_box_plot(csv_id):
     return Response(fig, mimetype='image/png')
 
 
+@csv_bp.route('/csv/<uuid:csv_id>/plot/pca', methods=['GET'])
+def get_pca_plot(csv_id):
+    csv_file = CSVFile.query.get_or_404(csv_id)
+    x_component_index = int(request.args.get('x_component_index', 0))
+    y_component_index = int(request.args.get('y_component_index', 1))
```

Comment:
why would I want to change these in my request?  to flip the axis?

Reply:
To choose which principal component is shown on each axis.  I think I remember Metaboanalyst having a feature like that.

Reply:
Edit: nevermind